CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 	4. KuhnsVI
 	5. KuhnsVII
 ### Changed
+- versions of `checkout` and `setup-python` github action
```

Comment:
Since `checkout` and `setup-python` aren't included in the main section of the library, I recommend replacing this line with `- Test system modified`.